packages/langium/src/languages/language-meta-data.ts

```diff
@@ -4,8 +4,17 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
+/**
+ * Metadata of a language.
+ */
 export interface LanguageMetaData {
     languageId: string;
     fileExtensions: readonly string[];
     caseInsensitive: boolean;
+    /**
+     * Mode used to optimize code for development or production environments.
+     * 
+     * In production mode, all lexer/parser validations are disabled.
```

Comment:
Maybe clarify that this refers specifically to the Chevrotain lexer/parser validations?
As I understand, this doesn't apply to the validations introduced in #1668